qiskit_machine_learning/algorithms/classifiers/qsvc.py

```diff
@@ -33,27 +38,25 @@ class QSVC(SVC):
 
     **Example**
 
-    .. code-block::
+    .. code-block::python
 
-        qsvc = QSVC(quantum_kernel=qkernel)
+        qsvc = QSVC(quantum_kernel=quant_kernel, kernel_trainer=qkt)
         qsvc.fit(sample_train,label_train)
         qsvc.predict(sample_test)
     """
 
-    def __init__(self, *args, quantum_kernel: Optional[QuantumKernel] = None, **kwargs):
-        """
+    def __init__(
+        self,
+        quantum_kernel: Optional[QuantumKernel] = None,
+        kernel_trainer: Optional[QuantumKernelTrainer] = None,
+        **kwargs,
+    ):
+        r"""
         Args:
-            quantum_kernel: QuantumKernel to be used for classification.
-            *args: Variable length argument list to pass to SVC constructor.
-            **kwargs: Arbitrary keyword arguments to pass to SVC constructor.
+            quantum_kernel:``QuantumKernel`` to be used for classification
+            kernel_trainer: ``QuantumKernelTrainer`` to be used for kernel optimization
```

Comment:
Can we elaborate here on the default behavior in the docstrings as I think its kinda different. For quantum_kernel, if one is not provided the algorithm spins up a default internally, and hence the getter will always get a valid instance despite one not being passed here. For the trainer, if nothing is passed, then there is simply no training done right. We don't have any trainer instance that does no training as some sort of default.